dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -633,3 +633,19 @@ ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host n
 ssl.client.handshake.error.CWPKI0824E.explanation=An error occurred during the SSL handshake.  Host name verification is enabled and server's identity can not be verified.  The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information.
 ssl.client.handshake.error.CWPKI0824E.useraction=When host name verification is enable the server's being accessed needs to be setup with a certificate contain the proper information needed to verify the server's identity.
 # -------------------------------------------------------------------------------------------------
+ssl.cacerts.error.CWPKI0825W=CWPKI0825W: The trustCacerts attribute is enabled and there is a failure trying to initialize the JDK default truststore.  The JDK truststore will not be used only the configured truststore will be used.  The [{0}] SSL configuration will be used without the JDK default truststore.  The exception returned is: [{1}].
```

Comment:
Change
"The trustCacerts attribute is enabled and there is a failure trying to initialize the JDK default truststore.  The JDK truststore will not be used only the configured truststore will be used.  The [{0}] SSL configuration will be used without the JDK default truststore.  The exception returned is: [{1}]."
To
"The trustCacerts attribute is enabled and the JDK default truststore failed to initialize. Only the configured truststore is used, and the JDK truststore is not used. The [{0}] SSL configuration is used without the JDK default truststore. The returned exception is: [{1}].

Should this read, "Without the JDK default truststore, the [{0}] SSL configuration is used", or is what's above correct?